code/modules/antagonists/traitor/contractor/datums/syndicate_contract.dm

```diff
@@ -436,7 +475,7 @@
 /datum/syndicate_contract/proc/handle_target_return(mob/living/M, turf/extraction_turf)
 	var/list/turf/possible_turfs = list()
 	for(var/turf/T in (list(extraction_turf) + contract.extraction_zone.contents))
-		if(!isspaceturf(T) && !isunsimulatedturf(T) && !is_blocked_turf(T))
+		if(T && !isspaceturf(T) && !isunsimulatedturf(T) && !is_blocked_turf(T))
```

Comment:
T can be null here?

Reply:
Admin topic `contractor_release` calls `contract.handle_target_return(M, null)` to return force the target to return to the default return location, which is Arrivals

Reply:
Sounds like `extraction_turf` should not be added in that case then